common/usr/share/MailScanner/perl/MailScanner/Message.pm

```diff
@@ -7683,9 +7689,14 @@ sub DisarmEndtagCallback {
         # https://github.com/MailScanner/v5/issues/229
         MailScanner::Log::DebugLog("DisarmLinkURL = $DisarmLinkURL");
         MailScanner::Log::DebugLog("linkurl = $linkurl");
-        if ($DisarmLinkURL =~ m/^ma[il]+to[:;]/i && $linkurl =~ /@/ ) {
-           my @list = split(/@/, $linkurl);
-           $linkurl = $list[1];
+        if ($DisarmLinkURL =~ m/^ma[il]+to[:;]/i ) {
```

Comment:
What use case does this have `[il]+` ? `malito:`??

Reply:
@Skywalker-11 This is Julian's original regex for the mailto, my best guess is to catch some accidental or deliberate typos.

Reply:
I thought about this some more and I think I'm going to go ahead and remove the funny regex in this context of use.  'mailto:' with any other deviation is not really a 'mailto:' so it therefore does not need to be disarmed unless I'm missing something...

Reply:
I have no idea what the original reason for this was, but I agree that if the href doesn't match /mailto:/i it won't be seen as a mailto: by a browser or mua and doesn't need disarming.

Reply:
There would also be no program registered for that URI so it won't do anything at all but throwing an error that the OS doesn't know any program to open it with

Reply:
https://github.com/MailScanner/v5/pull/339